MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -71,13 +72,15 @@ Use these conventions to indicate the status of each criterion.
   * _This is not applicable to libraries_
 * [ ] Sensitive and environment-specific information is not checked into git repository (6)
 * [ ] Written in a language and framework from the [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page[^1] (3, 5)
+    * For existing module evaluation: Consider "grandfathering" these modules in, e.g. by setting a date - only applicable to modules developed after ______.
```

Comment:
Rather than provide exceptions, note failures, will not necessarily result in action, take this note out.